pkg/apis/maistra/v2/runtime.go

```diff
@@ -193,6 +193,19 @@ type CommonPodRuntimeConfig struct {
 	// XXX: currently, this is only a global setting.  maybe only allow setting in global runtime defaults?
 	// +optional
 	PriorityClassName string `json:"priorityClassName,omitempty"`
+
+	// +optional
+	Volumes []Volumes `json:"volumes,omitempty"`
+}
+
+// Restrict Volumes to only support PVC 
+type Volumes struct {
+	// Volume's name. Must be a DNS_LABEL and unique within the pod
+	Name string `json:"name,omitempty"`
+
+	// PersistentVolumeClaim references the user's PVC in the same namespace.
+	// This volume finds the bound PV and mounts that volume for the pod
+	PersistentVolumeClaim corev1.PersistentVolumeClaimVolumeSource `json:"persistentVolumeClaim,omitempty"`
```

Comment:
@rcernich Do we also want to support more common volume types eg: `configMap`, `hostPath` and `emptyDir`?